pwndbg/commands/reload.py

```diff
@@ -8,23 +10,19 @@
 from pwndbg.commands import CommandCategory
 
 
-def rreload(module, mdict=None) -> None:
-    """Recursively reload modules."""
-    name = module.__name__
-
-    if mdict is None:
-        mdict = []
+def rreload(module, _exclude_mods=None) -> None:
+    """Recursively reload modules.
+    Impl based on https://stackoverflow.com/a/66661311/1508881"""
+    delete_folders = ["pwndbg"]
```

Comment:
Did you make this a list because you think we may want to reload other folders at some point?

Reply:
Original solution had it and I experimented with reloading gdb too, but it's probably not needed